src/flake8_pytest_style/plugins/unittest_assert.rs

```diff
@@ -11,6 +11,8 @@ use crate::ast::helpers::{create_expr, create_stmt};
 pub enum UnittestAssert {
     AlmostEqual,
     AlmostEquals,
+    CountEqual,
+    DictContainsSubset,
```

Comment:
`DictContainsSubset` exists, but is deprecated and hidden in [the unittest doc](https://docs.python.org/3/library/unittest.html#module-unittest):

https://github.com/python/cpython/blob/4b4e6da7b53f9eb2d3654d321f6acfbd599bd990/Lib/unittest/case.py#L1174